### PR TITLE
ISSUE<259> CHANGED NAMING OF API PATH VARIABLE FROM QUERY TO PATH

### PR DESCRIPTION
**Pull request by AE542**
# Pull Request

## What does this PR do?
Fixes #<259>
(https://github.com/meilisearch/meilisearch-swift/issues/259)

This commit will make a basic change to the variable name from query to path where it was called in the Documents Swift file.

I also added a dependency package (.package(url: "https://github.com/meilisearch/meilisearch-swift.git", from: "0.12.0") to the Package file at the top of the file in Xcode when trying to resolve some errors when trying to run meilisearch. If this was not necessary please tell me.

If there are any problems please contact me on here. I am a new contributor so I apologise if there are any issues with this PR. 

Thank you for your time.

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -10,7 +10,8 @@ let package = Package(
   dependencies: [
     // Support for dependabot for swift packages in dependabot
     // is in draft mode https://github.com/dependabot/dependabot-core/pull/3772
-    .package(url: "https://github.com/realm/SwiftLint.git", from: "0.43.1")
+    .package(url: "https://github.com/realm/SwiftLint.git", from: "0.43.1"),
+    .package(url: "https://github.com/meilisearch/meilisearch-swift.git", from: "0.12.0")
```
Comment (bidoubiwa) [inline Package.swift:14]: Why did you add this line?

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
Could you build and run the test locally to ensure your changes are working?

File: Package.swift (hunk 1 above)
Comment (AE542) [reply Package.swift:14]: That was my mistake sorry! I was trying to add the package for meiliisearch. I read the guidelines wrong. I just deleted that line a moment ago, while trying to fix this.

File: Sources/MeiliSearch/Documents.swift [hunk 2]
```diff
@@ -18,7 +18,7 @@ struct Documents {
     _ identifier: String,
     _ completion: @escaping (Result<T, Swift.Error>) -> Void)
   where T: Codable, T: Equatable {
-    let query: String = "/indexes/\(uid)/documents/\(identifier)"
+    let path: String = "/indexes/\(uid)/documents/\(identifier)"
     request.get(api: query) { result in
```
[the remark this answers is not in the document]
Comment (AE542) [reply Sources/MeiliSearch/Documents.swift:22]: Ok. I thought I did change it. That's also my fault! I'll do what I can to get it running properly. Thanks a lot for going through this.

**Comment by AE542** [comment]
> Could you build and run the test locally to ensure your changes are working?

Yes I'll do a run of the tests while making changes before another commit. Once again sorry for the trouble!

**Comment by AE542** [comment]
> Could you build and run the test locally to ensure your changes are working?

Hey bidoubiwa, I've made the changes from let query into let path in the Documents file, but I just have a question regarding the Integration Tests. If I run them via Xcode, they fail instantly with this error 

`2022-02-17 15:55:06.350508+0000 xctest[3134:145966] Task <58D62956-EE18-42C6-B662-EF22240984F1>.<1> HTTP load failed`

The regular Unit Tests work absolutely no problem but the integration tests all start failing. I presume it's a connection error but I'm not quite sure what the reasoning may be. Is there something I'm missing? Also apologies it's taken this long and thanks for your time.

**Comment by bidoubiwa** [comment]
Hey @AE542 no problem! Sorry I was on holiday. 

Are you running a Meilisearch instance while running the integrations tests ?

**Comment by bidoubiwa** [comment]
Hey @AE542 I would really like to merge your PR! Are you still working on it? If you have no time could you fix the conflicts so that I can merge it in another branch and finish your PR?
You'll still have the credit :)

**Comment by bidoubiwa** [comment]
I'm closing this issue as it is stale! If you'd like to work on it please tell me in which case we will re-open
